packages/features/ee/billing/repository/teamBillingData/PrismaTeamBillingRepository.ts

```diff
@@ -0,0 +1,29 @@
+import type { PrismaClient } from "@calcom/prisma";
+
+import type { ITeamBillingDataRepository } from "./ITeamBillingDataRepository";
+import { teamBillingSelect } from "./ITeamBillingDataRepository";
+
+export class PrismaTeamBillingDataRepository implements ITeamBillingDataRepository {
+  constructor(private prisma: PrismaClient) {}
+
+  /** Fetch a single team with minimal data needed for billing */
+  async find(teamId: number) {
```

Comment:
find -> _findUniqueOrThrow_. The name should reflect that it throws.

Similary for findBySubscriptionId below